Form/FieldsetType.php

```diff
@@ -8,49 +8,47 @@
 use Symfony\Component\Form\FormView;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class FieldsetType extends AbstractType {
-
+class FieldsetType extends AbstractType
+{
     /**
      * @param OptionsResolverInterface $resolver
      */
-    public function configureOptions ( OptionsResolver $resolver )
+    public function configureOptions(OptionsResolver $resolver)
     {
-        $resolver->setDefaults(array(
-            'legend'    => '',
-            'inherit_data'   => true,
-            'options'   => array(),
-            'fields'    => array(),
-            'label'     => false,
-        ))
-        ->addAllowedTypes(array(
-            'fields' => array('array', 'callable'),
-        ));
+        $resolver
+            ->setDefaults([
+                'legend' => '',
+                'inherit_data' => true,
+                'options' => array(),
+                'fields' => array(),
+                'label' => false,
+            ])
+            ->addAllowedTypes('fields', ['array', 'callable']);
```

Comment:
It's mostly a matter of consistency, either go short syntax everywhere, or long syntax everywhere, mixing feels sloppy, and I'm at fault here. :P